.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,19 +13,58 @@ For questions, the best place to get answers is on our [discussion forum](https:
 
 Please note: We take Boundary's security and our users' trust very seriously. If you believe you have found a security issue in Boundary, please responsibly disclose by contacting us at security@hashicorp.com. Our PGP key is available at [our security page](https://www.hashicorp.com/security/).
 
+If Boundary Desktop Client version >= 2.1.0
+
+Logs are found at:
+
+Linux: ~/.config/Boundary/logs/desktop-client.log
+MacOS: ~/Library/Logs/Boundary/desktop-client.log
+Windows: $env:USERPROFILE\AppData\Roaming\Boundary\logs\desktop-client.log
+
+We also have some debug logging initially which logs how long search requests take (only way to change this until log levels can be changed directly in settings page)
+
+Linux: BOUNDARY_DESKTOP_LOG_LEVEL=debug ./boundary-desktop
+MacOS: BOUNDARY_DESKTOP_LOG_LEVEL=debug ./Boundary.app/Contents/MacOS/Boundary
+Windows: $env:BOUNDARY_DESKTOP_LOG_LEVEL='debug'; .\Boundary.exe
```

Comment:
This isn't necessary anymore (and in fact doesn't work now), this was meant as info for us internally until we got the settings page released which we do now. You might want to change this to direct them to the settings page instead

Reply:
Gotcha. I will update this section.